pages/_document.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable @next/next/google-font-display */
+
+import Document, { Html, Head, Main, NextScript } from "next/document";
+
+// We use display=block to remove the jank
+
+class MyDocument extends Document {
+  render() {
+    return (
+      <Html>
+        <Head>
+          <link rel="preconnect" href="https://fonts.googleapis.com" />
+          <link
+            rel="preconnect"
+            href="https://fonts.gstatic.com"
+            crossOrigin="true"
+          />
+          <link
+            href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700;800&display=optional"
+            rel="stylesheet"
+          />
+        </Head>
+        <body>
+          <Main />
+          <NextScript />
+        </body>
+      </Html>
+    );
+  }
+}
+
+export default MyDocument;
```

Comment:
Why did you add this file? Did you see that any example of Nextra doesn't use `_document` file, why did you add it?